wordpress-mobile/gutenberg-mobile
# Fix focus PostTitle on new posts regression.

This PR fixes a regression where the event "setTitleFocus" was not handled properly on the JS side. This had the effect that the Title was not set to focus on new (empty) posts.

`gutenberg` side PR: https://github.com/WordPress/gutenberg/pull/17180
`WPiOS PR`: https://github.com/wordpress-mobile/WordPress-iOS/pull/12379

To test:
- Checkout the WPiOS PR.
- Open a new gutenberg post (no need to run metro).
- Check that the PostTitle is automatically focused.

The UI Tests on the WPiOS branch should also be passing now.
cc @JavonDavis 

Update release notes:
